.github/workflows/Tests.yaml

```diff
@@ -66,9 +70,23 @@ env:
   DIVE_IMAGE: 'wagoodman/dive:v0.12'
 
 jobs:
+  access-check:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v4
+      if: ${{ !inputs.is-workflow_call }}
+    - name: Validate access to triggered image
+      uses: ./.github/actions/validate-access
+      if: ${{ !inputs.is-workflow_call }}
+      with:
+        admin-only: true
+        image-path: ${{ inputs.oci-image-path }}
+        github-token: ${{ secrets.ROCKSBOT_TOKEN }}
```

Comment:
doesn't the GITHUB_TOKEN work?

Reply:
No, since the `team` attribute of an organization is private to internal uses only.